examples/dynamo/eg11/async_script.py

```diff
@@ -0,0 +1,39 @@
+def trans(psy):
+    '''sample transformation script to demonstrate the use of asynchronous
+    halo exchanges with overlapping compute and communication for the
+    most costly halo exchanges in the (current version of the) model
+
+    '''
+    from psyclone.transformations import \
+        Dynamo0p3RedundantComputationTrans, \
+        Dynamo0p3AsyncHaloExchangeTrans, \
+        MoveTrans
+
+    schedule = psy.invokes.invoke_list[0].schedule
+    schedule.view()
+
+    # This transformation removes the halo exchange associated with
+    # the grad_p field. This transformation is not be needed if
+    # annexed_dofs is set to True in the config file.
```

Comment:
Changed typo.

I'm not sure about checking annexed dofs. The code is still correct, we just change the bounds to halo(1) and that removes a halo exchange when annexed_dofs is False and does nothing when annexed dofs is True (apart from performing redundant computation into the level 1 halo rather than up to and including annexed dofs).

Reply:
OK. As long as it works in both cases I'm happy.